chainer/functions/pooling/roi_pooling_2d.py

```diff
@@ -50,9 +50,14 @@ class ROIPooling2D(function.Function):
 
     """RoI pooling over a set of 2d planes."""
 
-    def __init__(self, outh, outw, spatial_scale):
+    def __init__(self, outh, outw, spatial_scale, order='xy'):
         self.outh, self.outw = outh, outw
         self.spatial_scale = spatial_scale
+        if not isinstance(order, str) or order not in ['xy', 'yx']:
```

Comment:
Just checking `order not in ['xy', 'yx']` looks enough.

Reply:
OK, i will update.